docs/pages/versions/unversioned/guides/push-notifications.md

```diff
@@ -243,9 +251,10 @@ Each message must be a JSON object with the given fields:
 ```javascript
 type PushMessage = {
   /**
-   * An Expo push token specifying the recipient of this message.
+   * An Expo push token or an array of Expo push tokens specifying the recipient(s)
+   * of this message.
    */
-  to: string,
+  to: string | [string],
```

Comment:
```suggestion
  to: string | string[],
```

In some languages like TypeScript, `[string]`, means a tuple that contains one element that is a string (like `[string, number, number]`).